docs/standard/serialization/examples-of-xml-serialization.md

```diff
@@ -140,9 +140,9 @@ public class Address
   
 ```xml  
 <PurchaseOrder>  
-    <Address>  
+    <MyAddress>  
```

Comment:
I double-checked this attribute name typo (and the one below) by serializing it myself.